frontend/test/metabase/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -168,6 +169,20 @@ describe("scenarios > embedding > dashboard parameters", () => {
       cy.get(".ScalarValue")
         .invoke("text")
         .should("eq", "2,500");
+
+      // embedding with locale metabase#22561
+      if (!isEE) {
```

Comment:
We don't want to test enterprise embed as there's no *powered by* text to be searched which normally exists at the bottom left of the screen.
![image](https://user-images.githubusercontent.com/1937582/167815950-9c895658-0b6f-4d37-9286-8b75250b4c24.png)